protocol/v1/qbft/validation/signedmsg/quorum.go

```diff
@@ -0,0 +1,20 @@
+package signedmsg
+
+import (
+	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+
+	"github.com/bloxapp/ssv/protocol/v1/blockchain/beacon"
+)
+
+// HasQuorum checks if messages have a quorum.
+func HasQuorum(share *beacon.Share, msgs []*specqbft.SignedMessage) (quorum bool, t int, n int) {
```

Comment:
why new function? we have it on spec? if so, let's use it from there

Reply:
@nivBlox New function because spec checks for a quorum this way. We have it in spec, but it uses spec's share, so we cannot import it in ssv until ssv uses spec's share